### PR TITLE
Check Point - Playbook and Custom Connector

### PR DESCRIPTION
**Pull request by chkp-jguo**
Fixes #

## Proposed Changes

  - Add Check Point logo to logo directory
  - Add Check Point / Playbook folder to Solutions directory
  - Add Check Point / Playbook folder which contains Check Point custom connector and playbook

### CODE REVIEW

File: Solutions/Check Point/Playbooks/README.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+Common use cases include: 
+
+  1. Enable operation teams to automate common security functions such as creating objects, updating security policies, and schedule security policy updates to gateways. 
+  2. Fully integrate with any orchestration platforms for both on-prem or public cloud providers
+  3. Integrate with all leading SIEM/SOAR providers such as Azure Sentinel
+
+
+For more information see
+[Check Point Management API](https://sc1.checkpoint.com/documents/latest/APIs/#introduction~v1.7%20) and 
+[Logic App Overview](https://azure.microsoft.com/en-us/services/logic-apps/)
```
Comment (sarah-yo) [inline Solutions/Check Point/Playbooks/README.md:16]: Remove localisation "en-us" from URL

**Review by sarah-yo — CHANGES_REQUESTED** [review]
Hello, please can you do the following:

-  remove any localisation from your URLs (I have marked one example but please check your whole readme)
- Add a "Deploy to Azure Gov button and ARM template
- Update your "Deploy to Azure button image with the most recent version as can be seen on other readmes in this repo

Thanks!

File: Solutions/Check Point/Playbooks/README.md (hunk 1 above)
Comment (chkp-jguo) [reply Solutions/Check Point/Playbooks/README.md]: Removed localization - per https://github.com/Azure/Azure-Sentinel/pull/2052/commits/2b05dc45e52c19bc1a42eeaea62226fafa221843

**Comment by chkp-jguo** [comment]
> Hello, please can you do the following:
> 
> * remove any localisation from your URLs (I have marked one example but please check your whole readme)
> * Add a "Deploy to Azure Gov button and ARM template
> * Update your "Deploy to Azure button image with the most recent version as can be seen on other readmes in this repo
> 
> Thanks!

1. Removed localization in readme
2. Added deploy to azure gov and arm template
3. Updated deploy to azure button image

**Comment by chkp-jguo** [comment]
will submit a new one.

**Pull request closed without merging**